requirements.txt

```diff
@@ -1,7 +1,7 @@
 ops>=2.7.0
-jinja2==3.1.2
-netifaces==0.11.0
-jsonschema==4.19.1
-cryptography==41.0.4
+jinja2
+netifaces
+jsonschema
+cryptography
```

Comment:
I don't agree that we should remove the pinning here. As this will make us pulling always latest and make builds unpredictable as we don't know if a just released version may break us. Also renovate will come and pin them again.

Can you elaborate more of why you're doing this?

Reply:
The problem comes when we want to build charms for different series, 20.04 ships with python 3.8 which has issues with cryptography and takes a certain version for it. But when we want to build the same for 22.04 which comes with python 3.10 which supports a different crytography version. I will probably have to put markers here for python version as you mentioned renovate might look to pin this anyway.